mailbox/tools/quota-recompute/src/main/java/org/apache/james/mailbox/quota/task/RecomputeCurrentQuotasTaskAdditionalInformationDTO.java

```diff
@@ -50,29 +110,22 @@ public static AdditionalInformationDTOModule<RecomputeCurrentQuotasTask.Details,
     }
 
     private final String type;
-    private final long processedQuotaRoots;
-    private final ImmutableList<String> failedQuotaRoots;
+    private final List<RecomputeSingleQuotaComponentResultDTO> recomputeSingleQuotaComponentResults;
     private final Optional<RunningOptionsDTO> runningOptions;
     private final Instant timestamp;
 
     public RecomputeCurrentQuotasTaskAdditionalInformationDTO(@JsonProperty("type") String type,
-                                                              @JsonProperty("processedQuotaRoots") long processedQuotaRoots,
-                                                              @JsonProperty("failedQuotaRoots") ImmutableList<String> failedQuotaRoots,
+                                                              @JsonProperty("recomputeSingleQuotaComponentResults") List<RecomputeSingleQuotaComponentResultDTO> recomputeSingleQuotaComponentResults,
```

Comment:
This breaking change will break the distributed task manager. 

You NEED to enforce compatibility with the hold format and have deserialisation tests prooving that you handle the old format.

Here itis as simple as still accepting `processedQuotaRoots` and `failedQuotaRoots` as optionals and use it to populate a `recomputeSingleQuotaComponentResults` for the MAILBOX entry...